Translations/ko.lproj/preferences.strings

```diff
@@ -153,10 +153,10 @@
 "453.title" = "없음";
 
 /* Class = "NSButtonCell"; title = "Enable Notification Center integration"; ObjectID = "SOL-6U-5P9"; */
-"SOL-6U-5P9.title" = "Enable Notification Center integration";
+"SOL-6U-5P9.title" = "macOS 알림 센터 사용";
```

Comment:
Note that Keka 1.1.0 will still run in Mac OS X 10.9-10.11. Can you do it without that one?